cron/mozdefStateMappingTemplate.json

```diff
@@ -0,0 +1,278 @@
+{
+    "order": 0,
```

Comment:
Since most of this file isn't relevant to the mozdefstate index, can we instead just represent the options we care about (replicas/shards) in a dictionary object within the cron script?
Example:
```
mapping = {
    "settings": {
        "number_of_shards": 1,
         "number_of_replicas": 1
    }
}
es.create_index(index, mapping)
```

Reply:
Understood, since the mapping should be only a couple fields, would it be better to put that dictionary inside the cron script, instead of a separate file?

Reply:
Right now this only has mozdefstats fields, this mapping needs changed to include the mozdefhealth fields as well.
I'll add the additional fields and we'll see just how big it will be.

Also please note the sqs queues I'm adding will also be additional mappings. Let's decide once I have the finished mapping fleshed out.

Reply:
I've updated the mapping, and we should probably talk about this more once you've reviewed the sqs status PR, as any changes there may indicate a need for changes here - depending on whether we are ok with the queues being mapped dynamically to a field rather than a json blob.